api/test/singleton/CMakeLists.txt

```diff
@@ -0,0 +1,47 @@
+# Copyright The OpenTelemetry Authors
+# SPDX-License-Identifier: Apache-2.0
+
+include(GoogleTest)
+
+add_library(component_a STATIC component_a.cc)
+target_link_libraries(component_a opentelemetry_api)
+
+add_library(component_b STATIC component_b.cc)
+target_link_libraries(component_b opentelemetry_api)
+
+add_library(component_c SHARED component_c.cc)
```

Comment:
Should we disable `component_c` .. `component_f` on Windows?

Reply:
Not sure if I am missing something, but why is this unit test successful for these components on Windows as of now?

Reply:
> Not sure if I am missing something, but why is this unit test successful for these components on Windows as of now?

Because unit test is disabled on Windows in https://github.com/marcalff/opentelemetry-cpp/blob/9bfbfbe9d5d3ef462d0a4709e50070da49d561e2/api/test/singleton/CMakeLists.txt#L42

Reply:
ok, thanks. Yes, make sense to disable building them for windows then.

Reply:
Disabled the entire unit tests for windows.